drivers/sensor/current_amp/current_amp.c

```diff
@@ -73,6 +75,38 @@ static const struct sensor_driver_api current_api = {
 	.channel_get = get,
 };
 
+#ifdef CONFIG_PM_DEVICE
+static int pm_action(const struct device *dev, enum pm_device_action action)
+{
+	const struct current_sense_amplifier_dt_spec *config = dev->config;
+	int ret;
+
+	if (config->power_gpio.port == NULL) {
+		LOG_ERR("PM not supported");
+		return -ENOTSUP;
+	}
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		ret = gpio_pin_set_dt(&config->power_gpio, 1);
+		if (ret != 0) {
+			LOG_ERR("failed to set GPIO for PM resume");
```

Comment:
How about being explicit and `return ret` here and `return 0` at the end? Maybe bit more on the personal preference side but explicit exits are nice and unambiguous.

Reply:
Done